setup/update/sql/2.2.0-alpha1.sql

```diff
@@ -24,4 +24,14 @@ INSERT INTO  `hook_i18n` (`id`, `locale`, `title`, `description`, `chapo`) VALUE
 
 ALTER TABLE `admin_log` ADD `resource_id` INTEGER AFTER `resource` ;
 
+INSERT INTO `config` (`name`, `value`, `secured`, `hidden`, `created_at`, `updated_at`) VALUES
```

Comment:
look in previous sql files how we did the addition of new variable config. You have to retrieve first the last id an the insert in the table the new records.